frontend/src/components/NavBar/NavBar.jsx

```diff
@@ -41,20 +58,97 @@ import { UserAvatar } from './UserAvatar';
 
 const drawerWidth = 240;
 
+export const mobileCartNavItems = (itemCartCount, kitCartCount) => [
```

Comment:
`mobileCartNavItems` and other const has been moved to `frontend/src/components/NavBar/NavItems.jsx`